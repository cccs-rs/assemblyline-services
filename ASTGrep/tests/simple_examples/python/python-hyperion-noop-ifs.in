if 144647 > 3704800:
    _detectvar._add(_memoryaccess = 10582 * _detectvar.CallFunction)
elif 384468 < 4526755:
    Square(Multiply = -731 / -5155)._add(_memoryaccess = -1097 - _detectvar.CallFunction)

if 5 > 3:
    print("Hello")
    if some:
        print("inner")
    do("world")
elif 5 < 3:
    print("World")

if 5 > 8:
    print("Hello")
elif 8 > 9:
    print("World")

if 2>1:
    b2 = 0x80 | ((h2so3 >> 6) & 0x3F)

if 1<2<3:
    b3 = 0x80 | ((h2so3 >> 6) & 0x3F)

if 2==2:
    b2 = 0x80 | ((h2so3 >> 12) & 0x3F)

if 2<1:
    print("Hello")
else:
    print("World")

if not 37:
    llIIII11 = 5
else:
    llIIII11 = 37

if not False:
    c = 87
