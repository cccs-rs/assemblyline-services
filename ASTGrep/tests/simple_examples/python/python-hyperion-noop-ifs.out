Square(Multiply = 0.14180)._add(_memoryaccess = -1097 - _detectvar.CallFunction)


print("Hello")
if some:
    print("inner")
do("world")


pass


b2 = 0x80 | ((h2so3 >> 6) & 0x3F)


b3 = 0x80 | ((h2so3 >> 6) & 0x3F)


b2 = 0x80 | ((h2so3 >> 12) & 0x3F)


print("World")


llIIII11 = 37


c = 87
